scripts/build-color-variants.js

```diff
@@ -73,8 +66,15 @@ function isSemitransparent(color) {
   return /^(lighten|darken)/.test(color);
 }
 
-function isDark(color) {
-  return color === 'black' || /-dark$/.test(color);
+function isNotSuitableForForms(color) {
+  // Do not create form elements from values that are too light or too dark,
+  // for accessibility and to save space.
+  return color === 'black' || /^(darken10|lighten10)$/.test(color) || /(-dark|-light|-faint)$/.test(color);
+}
```

Comment:
What do you about think `-faint` for hover and active states? Should we keep those, or also remove them to encourage stronger contrast?